src/Arcus.Observability.Tests.Unit/Telemetry/ILoggerExtensionsTests.cs

```diff
@@ -803,6 +931,8 @@ public void LogServiceBusTopicDependencyWithDependencyMeasurement_ValidArguments
             Assert.Contains(startTime.ToString(FormatSpecifiers.InvariantTimestampFormat), logMessage);
             Assert.Contains(duration.ToString(), logMessage);
             Assert.Contains(isSuccessful.ToString(), logMessage);
+            // Check specifically on dependencyName
+            Assert.Contains("Azure Service Bus " + topicName, logMessage);
```

Comment:
Not a big fan of comments like this 😄 . Maybe if we move it to a dedicated variable, there's no need?
```suggestion
            Assert.Contains(isSuccessful.ToString(), logMessage);
            var dependencyName = topicName;
            Assert.Contains("Azure Service Bus " + dependencyName, logMessage);
```

Reply:
No problem 👍🏻  move the comments to dedicated variables, see: https://github.com/arcus-azure/arcus.observability/pull/229/commits/e821504de70e61733938d1a75efcee6498505887